tests/test_cli.py

```diff
@@ -427,3 +427,77 @@ def test_clean_no_task(tmp_empty):
 
     assert result.exit_code == 2
     assert "Error: Invalid value for 'FILENAME'" in result.output
+
+
+output_test = """\
+# ## first
+
+import fastparquet
+from pathlib import Path
+import pandas as pd
+
+j = open('file.txt', 'w')
+j.close()
+
+df = pd.DataFrame()
+df.to_csv('file_csv.csv')
+df.to_parquet('my.parquet')
+Path('write_text').write_text("stf")
+Path('write_byte').write_bytes('stf')
+
+
+"""
+
+
+def test_refactor_product_warning_for_output_files(tmp_empty):
```

Comment:
please parameterize this tests, it seems like you're doing the exact same in both